files/system/usr/share/ublue-os/just/70-secureblue.just

```diff
@@ -93,6 +93,19 @@ harden-flatpak:
         flatpak override --user --env=LD_PRELOAD=/var/run/host/usr/lib64/glibc-hwcaps/x86-64-v"$bestuarch"/libhardened_malloc.so
     fi
 
+# Harden a specified flatpak by preloading hardened_malloc (highest supported hwcap)
+harden-one-flatpak:
```

Comment:
instead of adding a separate command, why not just make the flatpak name an optional parameter of the existing command?

Reply:
good point. will do